### ISSUE TITLE
[rocminfo] hsa-runtime64 not found

### ISSUE THREAD

**Issue by acxz** [rocm-arch/rocm-arch#386]
As so:
```
==> Starting build()...
-- The C compiler identification is GNU 10.2.0
-- The CXX compiler identification is GNU 10.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:60 (find_package):
  By not providing "Findhsa-runtime64.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "hsa-runtime64", but CMake did not find one.

  Could not find a package configuration file provided by "hsa-runtime64"
  (requested version 1.0) with any of the following names:

    hsa-runtime64Config.cmake
    hsa-runtime64-config.cmake

  Add the installation prefix of "hsa-runtime64" to CMAKE_PREFIX_PATH or set
  "hsa-runtime64_DIR" to a directory containing one of the above files.  If
  "hsa-runtime64" provides a separate development package or SDK, be sure it
  has been installed.


-- Configuring incomplete, errors occurred!
See also "/home/acxz/vcs/git/github/rocm-arch/rocm-arch/rocminfo/src/CMakeFiles/CMakeOutput.log".
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by tpkessler** [comment]
I cannot reproduce this in a clean chroot.

**Comment by acxz** [comment]
let me try again with the updated packages

**Comment by acxz** [comment]
yep not an issue
